public/app/features/dashboard-scene/edit-pane/DashboardEditableElement.tsx

```diff
@@ -42,12 +43,18 @@ export class DashboardEditableElement implements EditableDashboardElement {
               return <DashboardDescriptionInput dashboard={dashboard} />;
             },
           })
+        )
+        .addItem(
+          new OptionsPaneItemDescriptor({
+            title: t('dashboard.layout.common.layout', 'Layout'),
+            render: function renderTitle() {
```

Comment:
Nit but i think we can remove the named function and go with some simple like `() => <DashboardLayoutSelector layoutManager={body} />;`